bin/storm

```diff
@@ -82,6 +82,8 @@ CONFFILE = ""
 JAR_JVM_OPTS = shlex.split(os.getenv('STORM_JAR_JVM_OPTS', ''))
 JAVA_HOME = os.getenv('JAVA_HOME', None)
 JAVA_CMD = 'java' if not JAVA_HOME else os.path.join(JAVA_HOME, 'bin', 'java')
+if JAVA_HOME and not os.path.exists(JAVA_CMD):
+    print "WARNING:  Check JAVA_HOME.  Could not find java at %s." % JAVA_HOME
```

Comment:
I personally would rather have storm exit with an error here instead of just a warning.